ZalikovkaC#/Program.cs

```diff
@@ -10,28 +10,30 @@ class Program
     {
         static void Main(string[] args)
         {    
-            int higerMath, physics, informatics, history, TCA, ecology, english, PE;
+            int higherMath, physics, informatics, history, TDA, ecology, english, PE;
             Console.WriteLine("Enter the rating for higher math:");
-            higerMath = int.Parse(Console.ReadLine());
+            higherMath = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter an assessment in physics:");
             physics = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter an assessment in computer science:");
             informatics = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter an estimate from history:");
             history = int.Parse(Console.ReadLine());
-            Console.WriteLine("Enter the TCA score:");
-            TCA = int.Parse(Console.ReadLine());
+            Console.WriteLine("Enter the TDA score:");
+            TDA = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter environmental assessment");
             ecology = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter an assessment in English:");
             english = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter the assessment in physical education:");
             PE = int.Parse(Console.ReadLine());
-            double numeral = (Coeficients.higherMath * higerMath + Coeficients.physics * physics + Coeficients.informatics * informatics
-                + Coeficients.history * history + Coeficients.tca * TCA +Coeficients.english*english
+            double numeral = (Coeficients.higherMath * higherMath + Coeficients.physics * physics + Coeficients.informatics * informatics
+                + Coeficients.history * history + Coeficients.TDA * TDA +Coeficients.english*english
                 + Coeficients.ecology * ecology + Coeficients.PE * PE);   
             double rating;
-            rating = (90.0 * numeral)/ (Coeficients.sum * 100);
+            const double maxAssessmentOfDiscipline = 90;
+            const int maxAssessmentOfRating = 100;
+            rating = (maxAssessmentOfDiscipline * numeral)/ (Coeficients.sum * maxAssessmentOfRating);
```

Comment:
I have already pointed that "Can we divide this formula into parts for instance to calculate the numerator and denominator separately to increase readability?" 
Please answer to every my comment, was it fixed or not and why not.